src/Microsoft.OData.Edm/GlobalSuppressions.cs

```diff
@@ -243,6 +243,7 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1810:InitializeReferenceTypeStaticFieldsInline", Scope = "member", Target = "Microsoft.OData.Edm.Vocabularies.V1.CapabilitiesVocabularyModel.#.cctor()")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1810:InitializeReferenceTypeStaticFieldsInline", Scope = "member", Target = "Microsoft.OData.Edm.Vocabularies.V1.CoreVocabularyModel.#.cctor()")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1810:InitializeReferenceTypeStaticFieldsInline", Scope = "member", Target = "Microsoft.OData.Edm.Vocabularies.Community.V1.AlternateKeysVocabularyModel.#.cctor()")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1810:InitializeReferenceTypeStaticFieldsInline", Scope = "member", Target = "Microsoft.OData.Edm.Vocabularies.Community.V1.AuthorizaationVocabularyModel.#.cctor()")]

```

Comment:
typo: **Authorizaation**VocabularyModel.#.cctor() -> **Authorization**VocabularyModel.#.cctor()

Will it be non-trivial to correct the name?

Reply:
Thanks.